### PR TITLE
Created class for command utilities and updated pom.xml

### PR DESCRIPTION
**Pull request by Dacaspex**
### What is this?
Refactored part of the code that is responsible for parsing command strings. Created a static utility class for this. 

Added Sonar to the pom.xml for coverage reports from JUnit. 

Please do not merge yet (I want to setup Travis first)

### CODE REVIEW

**Comment by Dacaspex** [comment]
Possible wrong configuration, see https://stackoverflow.com/questions/45573102/sonarcloud-failure-with-travis-maven-github

**Comment by Dacaspex** [comment]
Please use the "Squash and commit" merge strategy

File: src/main/java/com/hansque/commands/CommandStringUtil.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        String[] parts = commandString.split(" ");
+        parts[0] = aliases.getOrDefault(parts[0], parts[0]);
+
+        return String.join(" ", parts);
+    }
+
+    public static String getModuleFromCommandString(String commandString) {
+        String[] parts = commandString.split(":");
+        if (parts.length < 2) {
+            throw new IllegalArgumentException("The command string must contain at least one colon to identify the ");
```
Comment (Extremelyd1) [inline src/main/java/com/hansque/commands/CommandStringUtil.java]: Unfinished sentence?

**Review by Extremelyd1 — CHANGES_REQUESTED** [review]

File: src/main/java/com/hansque/commands/CommandStringUtil.java (hunk 1 above)
Comment (Dacaspex) [reply src/main/java/com/hansque/commands/CommandStringUtil.java]: Fixed

**Review by Extremelyd1 — APPROVED** [review]

**Pull request merged**